.github/workflows/sycl-nightly.yml

```diff
@@ -50,6 +50,14 @@ jobs:
             target_devices: ext_oneapi_hip:gpu
             tests_selector: e2e
 
+          - name: Intel Build Tests Only
```

Comment:
I don't think we really need this change just yet. IMO, we can develop the feature and limit the testing to manually-triggered jobs.

Reply:
Ok, i've removed the nightly workflow